packages/admin-panel/src/widgets/InputField/FileUploadField.jsx

```diff
@@ -3,26 +3,12 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { FileUploadField as BaseFileUploadField } from '@tupaia/ui-components';
 
-export const FileUploadField = ({
-  onChange,
-  name,
-  label,
-  helperText,
-  textOnButton,
-  showFileSize,
-  maxSizeInBytes,
-  initialFileName = null,
```

Comment:
As far as I can tell, this wasn’t actually doing anything. Even in `EditSurveyPage` (the only usage that provides an `initialFileName`), it wasn’t actually displayed. The `FileUploadField` just always accurately reflected what files were(n’t) selected

Reply:
Turns out DataTrak might use this (trusted my editor too much 🥲). I’ll figure out what to do about this